ouroboros-consensus-diffusion/src/Ouroboros/Consensus/Node.hs

```diff
@@ -678,17 +678,17 @@ stdChainSyncTimeout = do
     -- f = 0.05
     -- The timeout is randomly picked per bearer to avoid all bearers
     -- going down at the same time in case of a long streak of empty
-    -- slots. TODO: workaround until peer selection governor.
-    mustReplyTimeout <- Just <$> randomElem [90, 135, 180, 224, 269]
+    -- slots.
+    -- To avoid global synchronosation the timeout is picked uniformly
+    -- from the interval 135 - 269, corresponds to the a 99.9% to
+    -- 99.9999% thresholds.
+    -- TODO: workaround until peer selection governor.
```

Comment:
do you remember what the TODO is about?

Reply:
No, but the comment was written by me in 7c0afa95456dcc9f8e8516aa81fd418f94a2b8f5 .

If there should be a TODO I think it should be about moving from a static per connection timeout to a new random timeout every time chainsync ends up in the can-wait state (#4244)

Reply:
Let's update the todo then.

Reply:
Done.